src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        int a = random.nextInt(10 + 1);
+        int b = random.nextInt(10 + 1);
+        int h = random.nextInt(10 + 1);
+        ColorSupplier color = new ColorSupplier();
+        String randomColor = color.getRandomColor();
+        Circle circle = new Circle(a, randomColor);
+        Square square = new Square(a, randomColor);
+        Rectangle rectangle = new Rectangle(a, b, randomColor);
+        RightTriangle rightTriangle = new RightTriangle(a, b, randomColor);
+        IsoscelesTrapezoid isoscelesTrap = new IsoscelesTrapezoid(a, b, h, randomColor);

```

Comment:
You create 5 objects when you need only one? Maybe decide what exactly you will create first?)))
Use switch here